RELEASING.md

```diff
@@ -0,0 +1,42 @@
+# Releasing
+
+In order to create and publish the new version of maven artifact you have to have an account on http://developer.jboss.org and have the permissions for releasing.
+
+Next step is using the same credentials and saving them into your Maven settings. 
+
+Here is the minimal example of the `~/.m2/settings.xml`:
+
+```xml
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0
+                          https://maven.apache.org/xsd/settings-1.0.0.xsd">
+  <servers>
+    <server>
+      <id>jboss-nexus-thirdparty</id>
+      <username>your_username</username>
+      <password>your_password</password>
+    </server>
+  </servers>
+</settings>
+```
+
+If you don't want to save your credentials in plain-text, you may want to use the [password encryption](http://maven.apache.org/guides/mini/guide-encryption.html).
+
+> Note that server id (`jboss-nexus-thirdparty`) has to match with the server id, defined in the [pom.xml](https://github.com/radanalyticsio/streaming-amqp/blob/55f33a3fe8e1522573d536c147dbd88db5e7948c/pom.xml#L23).
+
```

Comment:
Can we use a simpler link to the POM file even because it's referring to a specific commit and not the master

Reply:
this is on purpose, the link contains the information about the line number and if it's in this form:
https://github.com/radanalyticsio/streaming-amqp/blob/master/pom.xml#L23
, it would break in the future if the line numbers change. This way it's reproducible. Other option would be using simple `./pom.xml` without the line number info.

Reply:
Yeah if it's not a problem for you I'd prefer to have the pom.xml reference without the line number.

Reply:
ok